scripts/EN/Harry

```diff
@@ -1680,20 +1680,20 @@ CleanScreen(); // 0x1c9a: 00000064
 LoadBackground(0x1ce); // 0x1c9e: 00001069
 DrawBackground(); // 0x1ca4: 00000065
 LoadSecondEffect(0xd, 0x1, 0x0); // 0x1ca8: 0000306e
-ShowText((1065)"After being left alone in the cafe I gazed at the handgun Cybil gave me for a little while. And then I thought about Cheryl, who had disappeared in this town."); // 0x1cb2: 0000100c
+ShowText((1065)"After being left alone in the cafe I gazed at the handgun Cybil gave me for a little while.¥n¥nW2And then I thought about Cheryl, who had disappeared in this town."); // 0x1cb2: 0000100c
 HandleInput(); // 0x1cb8: 00000010
-ShowText((1066)"The figure I saw in the fog... That was definitely Cheryl. But she didn't answer my calls, and she was quite possibly moving away from me?"); // 0x1cbc: 0000100c
+ShowText((1066)"The figure I saw in the fog…¥nW3That was definitely Cheryl.¥nBut she didn't answer my calls, W2and she was quite possibly moving away from me?"); // 0x1cbc: 0000100c
 HandleInput(); // 0x1cc2: 00000010
-ShowText((1067)"I don't understand... I am getting a bit mixed up. Still, the one thing I know for sure is that Cheryl has vanished in this town."); // 0x1cc6: 0000100c
+ShowText((1067)"I don't understand…¥nW2I am getting a bit mixed up.¥nW2Still, W1the one thing I know for sure is that W1Cheryl has vanished in this town."); // 0x1cc6: 0000100c
 HandleInput(); // 0x1ccc: 00000010
-ShowText((1068)"All I can do is look for her. I may not have any knowledge of the surroundings or any clues, but I won't find Cheryl if I stay here. I stood up energetically and placed my hand on the cafe door."); // 0x1cd0: 0000100c
+ShowText((1068)"All I can do is look for her.¥nW2I may not have any knowledge of the surroundings or any clues, but I won't find Cheryl W1if I stay here.¥nW2I stood up energetically W1and placed my hand on the cafe door."); // 0x1cd0: 0000100c
 HandleInput(); // 0x1cd6: 00000010
 PlaySoundFx(0xd); // 0x1cda: 00001072
-ShowText((1069)"(JiJiJi...JiJi...JiJiJi.........)"); // 0x1ce0: 0000100c
+ShowText((1069)"(Hisss…W2Hiss…W2Hisss………)"); // 0x1ce0: 0000100c
```

Comment:
Radio static onomatopoeia.

Used Hiss for English, with additional s letters to match the longer Japanese bits.